ios/app/brave_main_delegate.mm

```diff
@@ -52,8 +52,6 @@
 
 void BraveMainDelegate::BasicStartupComplete() {
   auto* command_line(base::CommandLine::ForCurrentProcess());
-  command_line->AppendSwitch(switches::kDisableEnterprisePolicy);
```

Comment:
@kylehickinson @Brandon-T: the above switch has been removed from Chromium. I added a patch/overrides in `[iOS] WIP: disable-enterprise-policy flag was removed.` commit (currently https://github.com/brave/brave-core/pull/13498/commits/bba65256fcdc48fac5516d496f74ec619fce04c4) that look to me like they would be sufficient, but I can't verify. Would be great if you could check at some point.

Reply:
@mkarolin Anything we need to check for in particular aside from no crashes at launch/setup?

Reply:
@kylehickinson looks like originally the switch was added for sync to work (in https://github.com/brave/brave-core/pull/6722), so probably some sync-related activities 😄

Reply:
Tested in isolation using `1.42.26`/`104.0.5112.20`:

![IMG_5377](https://user-images.githubusercontent.com/529104/175966276-c490a4cf-4ebb-4286-8292-4f6d5c77f0a7.PNG)

Seems like everything's OK. cc @Brandon-T to verify the sync internals screenshot